### PR TITLE
Add examples for mark and unmark

### LINKED ISSUE #122: [PE-D][Tester E] UG should contain examples for mark
The constraints for the `mark` command are good but there should be examples here, similar to `delete` that illustrate the functionality of the `mark` command.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: pjanthony2001/ped#9

### LINKED ISSUE #123: [PE-D][Tester E] UG should contain examples for unmark
The `unmark` command should have examples as well similar to the `delete` command. Some examples like `unmark 1` and screenshots would help illustrate the expected UI changes.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: pjanthony2001/ped#10

### PR DESCRIPTION
**Pull request by AjayArvind2207**
Fixes #123 
Fixes #122 

Add relevant examples for mark and unmark in the UG to make it easier to follow

### CODE REVIEW

**Comment by Xilef121** [comment]
LGTM, but you can consider adding images do better illustrate how the commands work

File: docs/UserGuide.md [hunk 1]
```diff
@@ -279,6 +279,11 @@ Format: `mark INDEX`
 * The index refers to the index number shown in the currently displayed company list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
+  Examples:
+* `list` followed by `mark 2` marks the 2nd company in the intern book.
+* `find Google` followed by `mark 1` marks the 1st company in the results of the `find` command.
```
Comment (jasperng-nus) [inline docs/UserGuide.md:284]: i think the could be consistent with the unmark, at the end could also include, "if it is unmarked"

**Review by jasperng-nus — COMMENTED** [review]
Overall LGTM, except for some minor inconsistency with mark and unmark examples and perhaps you could add some images of the examples to better guide the user

**Review by jasperng-nus — COMMENTED** [review]
LGTM!

**Pull request merged**